### PR TITLE
Extend the Undefined implementation to deal with dictionaries.

### PR DESCRIPTION
**Pull request by pelson**
Ping @jakirkham

### CODE REVIEW

**Comment by jakirkham** [comment]
Yeah, that looks cleaner. :+1:

**Comment by jakirkham** [comment]
Looks like this [broke]( https://travis-ci.org/conda-forge/conda-forge.github.io/builds/123667962 ) Travis. :disappointed: Will revert for now and we can try again later.

**Comment by jakirkham** [comment]
See reversion PR ( https://github.com/conda-forge/conda-forge.github.io/pull/105 ).

File: scripts/update_teams.py [hunk 1]
```diff
@@ -107,9 +83,7 @@ def create_team(org, name, description, repos):
     env = jinja2.Environment(undefined=NullUndefined)
 
     with open(recipe) as fh:
-        environ = SilentDict()
-        environ.undefined = ""
-        contents = env.from_string(''.join(fh)).render(environ=environ)
+        contents = env.from_string(''.join(fh)).render()
```
Comment (jakirkham) [inline scripts/update_teams.py]: I think all that was missing is we need to make `environ` an empty dictionary. Currently it is just undefined. So, that should fix it. Unfortunately, I don't have the time to play with it now.

Comment (pelson) [inline scripts/update_teams.py]: @jakirkham - we *really* need to stop using ``environ``. It is the wrong solution to the problem.
The whole VC thing is a mess, and it is driving us down a path we shouldn't be taking. Looking forwards to finding a more long-term solution this week 😄

**Pull request merged**